Client/Frontend/Browser/TabManager.swift

```diff
@@ -859,6 +859,9 @@ extension TabManager: WKNavigationDelegate {
 
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
         hideNetworkActivitySpinner()
+        if let tab = self[webView] {
+            TabEvent.post(.didChangeContentBlocking, for: tab)
```

Comment:
Should this be called `didChangeContentBlocking` or `didFinish` to be more generic?

Reply:
Prefer: `.didFinishNavigation`

Reply:
This won't work (not that I am aware of anyway). didFinish is for the main frame, web app frameworks hit this spot very quickly, then content is loaded async. I am skeptical we will find a tab event (or multiple events) to tie the icon to, I think the notifications will have to originate from the content blocking classes to reflect state changes reliably.

Have the setter for the TPPageStats send a notification that stats changes (and pass the stats)?